### DISCUSSION TITLE
Can Fooocus-Ruined run ComfyUI Workflows JSON file

### DISCUSSION THREAD

**Discussion by kengun2077** [runew0lf/RuinedFooocus#198] [General]
We know that  Fooocus-Ruined launch.py

git_repos = [
 {
 "name": "ComfyUI",
 "path": "ComfyUI",
 "url": "https://github.com/comfyanonymous/ComfyUI",
 "hash": "935ae153e154813ace36db4c4656a5e96f403eba",
 "add_path": "ComfyUI",
 },

]

so,  Fooocus-Ruined may run  ComfyUI Workflows JSON file

**Comment by yownas** [comment]
Sorry, no. Even if RF used Comfy as backend there is no good way to squeeze comfy workflows into our user interface.
If you want to use workflows I'd recommend SwarmUI https://github.com/mcmonkeyprojects/SwarmUI

**Comment by kengun2077** [comment]
ComfyUI Online

https://www.runcomfy.com/comfyui-web

https://comfyuiweb.com/
![1](https://github.com/user-attachments/assets/3d4d7585-7ff9-472f-a1fd-20fc82c8707f)
![2](https://github.com/user-attachments/assets/f297d6c5-889f-4271-980f-8dab5e49845e)
